src/Microsoft.Diagnostics.Runtime/ClrThread.cs

```diff
@@ -30,6 +30,8 @@ internal ClrThread(IClrThreadHelpers helpers, ClrRuntime runtime, ClrAppDomain?
             LockCount = data.LockCount;
             State = (ClrThreadState)data.State;
             _exceptionHandle = data.LastThrownObjectHandle;
+            this.IsFinalizer = isFinalizer;
+            this.IsGc = isGc;
```

Comment:
One nitpick, we don't use `this.` in any place here.  All members/properties have to be unambiguous.  Members start with `_` statics with `s_`, locals with lowercase, properties with uppercase.